src/coordinator/roles.js

```diff
@@ -161,6 +257,94 @@ class Roles {
       .get(`${this._baseUrl}/organizations/${organizationId}/roles`)
       .then((roles) => toCamelCase(roles));
   }
+
+  /**
+   * Remove an application from a role
+   *
+   * API Endpoint: '/applications/:applications_id/roles/:roleId'
+   * Method: DELETE
+   *
+   * @param {string} organizationId The ID of the organization
+   * @param {string} roleId The UUID formatted ID of the role
+   * @param {string} applicationId The UUID formatted ID of the application
+   *
+   * @returns {Promise}
+   * @fulfill {undefined}
+   * @reject {Error}
+   *
+   * @example
+   * contxtSdk.roles
+   *   .removeApplication('4f0e51c6-728b-4892-9863-6d002e61204d', '36b8421a-cc4a-4204-b839-1397374fb16b', '007ca9ee-ece7-4931-9d11-9b4fd97d4d58')
+   *   .then((roleApplication) => console.log(roleApplication))
```

Comment:
It's usually a 204 on delete, so I'd remove this from the docs

Reply:
thx...done